### PR TITLE
Update file-system.md

### PR DESCRIPTION
**Pull request by ajithpv**
Modify the file system section slightly with typo fix and sentence update.

### CODE REVIEW

File: en/overview/file-system.md [hunk 1]
```diff
@@ -3,20 +3,20 @@
 ```
 # File system
 
-LittlevGL has File system abstraction module which enables to attache any type of file systems.
-The file system are identified by a letter. 
-For example if the SD card is associated with letter`'S'` a file can be reached like `""S:path/to/file.txt`.
+LittlevGL has a 'File system' abstraction module that enables you to attach any type of file systems.
+The file system is identified by a letter.
+For example, if the SD card is associated with the letter `'S'`, a file can be reached like `""S:path/to/file.txt`.
```
Comment (kisvegabor) [inline en/overview/file-system.md]:
```suggestion
For example, if the SD card is associated with the letter `'S'`, a file can be reached like `"S:path/to/file.txt"`.
```

**Comment by kisvegabor** [comment]
@ajithpv 
Thank you! Let's wait for @embeddedt's review before merging.

May I ask what is your motivation to read the docs page by page? Do you use LittelvGL for education or just want to contribute this way? Anyway, we are very grateful for the reviews and updates. :slightly_smiling_face:

**Comment by ajithpv** [comment]
> May I ask what is your motivation to read the docs page by page? Do you use LittelvGL for education or just want to contribute this way? Anyway, we are very grateful for the reviews and updates. slightly_smiling_face

I'm learning littlevGL during my free time (I'm a full-time embedded developer who works on the processor side). When I start with the document, I saw some typos and grammar issues. Hence, I thought of correcting it but, due to the time limitation, I'm doing it by page by page now. :slightly_smiling_face:

File: en/overview/file-system.md (hunk 1 above)
Comment (embeddedt) [reply en/overview/file-system.md]: Change committed.

**Comment by embeddedt** [comment]
> I saw some typos and grammar issues. Hence, I thought of correcting it but, due to the time limitation, I'm doing it by page by page now.

We appreciate you taking the time to correct it!

**Comment by kisvegabor** [comment]
> I'm learning littlevGL during my free time (I'm a full-time embedded developer who works on the processor side). When I start with the document, I saw some typos and grammar issues. Hence, I thought of correcting it but, due to the time limitation, I'm doing it by page by page now. slightly_smiling_face

I see, awesome :slightly_smiling_face:

**Pull request merged**